source/_integrations/switchbee.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: SwitchBee
+description: Instructions for how to integrate SwitchBee accessories within Home Assistant.
+ha_category:
+  - Switch
+ha_release: 2022.10
+ha_iot_class: local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@jafar-atili'
+ha_domain: switchbee
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+[SwitchBee](https://www.switchbee.com), is an innovation company making smart homes more accessible and affordable to any household environment. 
+
+There is currently support for the following device types within Home Assistant:
+
+- Switch 
+- Timed Power Switch (Boiler)
+- Group Switch
+- Timed Switch
+
+Supported devices will be discovered after the `SwitchBee` integration is configured
+
+## Preliminary Steps
+
+You will need to provide the following parameters in order to set up the `SwitchBee` integration:
+
+- Central Unit IP
+- Username
+- Password
+- Initialize switches as light entities (False by default)
+
+
+## SwitchBee Options
+
+- `Devices to include`: Select the devices (by type) you wish to control from Home Assistant.
```

Comment:
That is an odd option? And sounds against our implementation guidelines.

Integrations should not provide means to enable/disable devices. Home Assistant should be handling that from the core itself.

Reply:
Its not really include/exclude platform

For instance, Cover can be a physical wall relay and also can be virtual Somfy Device. Same for switch (Boiler/virtual/wall switch)

Instead of polling all the data from the Central Unit, user should be able to decide what is needed and what is not.

This will save us fetching and reading those devices states from the Central Unit itself.

These are my thoughts.

Reply:
> Instead of polling all the data from the Central Unit, user should be able to decide what is needed and what is not.

That is a decision the user can already make and should not be made or implemented by an integration itself.